frontend/src/screens/apps/AppCreated.tsx

```diff
@@ -61,20 +61,24 @@ function AppCreatedInternal() {
     }
     // dispatch a success event which can be listened to by the opener or by the app that embedded the webview
     // this gives those apps the chance to know the user has enabled the connection
-    const nwcEvent = new CustomEvent("nwc:success", { detail: {} });
+    const nwcEvent = new CustomEvent("nwc:success", {
+      detail: {
+        nostrWalletConnectUrl: pairingUri,
+      },
+    });
     window.dispatchEvent(nwcEvent);
 
     // notify the opener of the successful connection
     if (window.opener) {
       window.opener.postMessage(
         {
           type: "nwc:success",
-          payload: { success: true },
+          nostrWalletConnectUrl: pairingUri,
```

Comment:
can we avoid this. 
doesn't the client create the secret? and thus the pairing Uri does not need to be passed back in full.

places where the paring secret is passed should be avoided as it could be a security issue. (e.g. in what hack somebody could listen to this postMessage)

also best to avoid making changes with sensitive data in a non-senstive pr.

Reply:
It does not contain the secret.
But we could return just the `relayUrl` and `walletServicePubkey` instead, to be very clear. Maybe this is better anyway, since the app has to construct its own client.

Reply:
Because this is part of the deeplink flow, we would need to create a spec around this, and by doing it this way we force all other wallet services to use a client-created secret. But that's probably for the best.

Reply:
I think passing the pairing uri without the secret param is good, would also help if some clients need lud16 for some reason (or if anything else is added in future to the pairing uri, instead of passing it separately)

Reply:
Reusing connections is a flaw and this also opens up security issues.
If the wallet doesn't support client-created secrets, then the user should use the standard copy paste flow.